aaelf32/aaelf32.rst

```diff
@@ -0,0 +1,3289 @@
+  +------+---------------------------------+----------+--------------------------------------------------------------------------+----------------+
+  | 11   | :code:`R_ARM_THM_PC8`           | Yes      | | LDR(3)/LDR(literal), ADD(5)/ADR                                        | X & 0x3FC      |
+  +------+---------------------------------+----------+--------------------------------------------------------------------------+----------------+
+  | 52   | :code:`R_ARM_THM_JUMP6`         | Yes      | CBZ, CBNZ                                                                | X & 0x7E       |
+  +------+---------------------------------+----------+--------------------------------------------------------------------------+----------------+
+  | 102  | :code:`R_ARM_THM_JUMP11`        | Yes      | B(2)/B                                                                   | X & 0xFFE      |
+  +------+---------------------------------+----------+--------------------------------------------------------------------------+----------------+
+  | 103  | :code:`R_ARM_THM_JUMP8`         | Yes      | B(1)/B<cond>                                                             | X & 0x1FE      |
+  +------+---------------------------------+----------+--------------------------------------------------------------------------+----------------+
+  | 132  | :code:`R_ARM_THM_ALU_ABS_G0_NC` | No       | | ADD(2)/ADD (immediate, Thumb, 8-bit immediate), MOV(1)/MOV (immediate) | X & 0x000000FF |
+  +------+---------------------------------+----------+--------------------------------------------------------------------------+----------------+
+  | 133  | :code:`R_ARM_THM_ALU_ABS_G1_NC` | No       | | ADD(2)/ADD (immediate, Thumb, 8-bit immediate), MOV(1)/MOV (immediate) | X & 0x0000FF00 |
+  +------+---------------------------------+----------+--------------------------------------------------------------------------+----------------+
+  | 134  | :code:`R_ARM_THM_ALU_ABS_G2_NC` | No       | | ADD(2)/ADD (immediate, Thumb, 8-bit immediate), MOV(1)/MOV (immediate) | X & 0x00FF0000 |
+  +------+---------------------------------+----------+--------------------------------------------------------------------------+----------------+
+  | 135  | :code:`R_ARM_THM_ALU_ABS_G3`    | No       | | ADD(2)/ADD (immediate, Thumb, 8-bit immediate), MOV(1)/MOV (immediate) | X & 0xFF000000 |
+  +------+---------------------------------+----------+--------------------------------------------------------------------------+----------------+
+
+Static Thumb32 relocations
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Relocations for 32-bit Thumb instructions are shown in the following table.
+With the exception of R_ARM_THM_CALL, these relocations are only applicable to
+32-bit Thumb instructions.
+
+.. _Static Thumb-32 instruction relocations:
+
+.. table:: Static Thumb-32 instruction relocations
+  :class: longtable
```

Comment:
as described above on the version history table, please use a class to specify the width and no longtable